recipes/gssdp/gssdp.inc

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "GSSDP, A GObject-based API for handling resource discovery and announcement over SSDP."
+LICENSE = "LGPL-2.0-or-later"
+
+inherit library 
+inherit meson
+inherit pkgconfig
```

Comment:
You can drop this now that meson.oeclass does this for you.

Reply:
Removed